src-docs/src/views/inline_edit/inline_edit_controlled.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+
+import { EuiInlineEditText } from '../../../../src';
+
+export default () => {
+  const [inlineEditValue, setInlineEditValue] = useState('Hello World!');
+
+  const inlineEditOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
optional nit: we can destructure this in the React import line
```suggestion
  const inlineEditOnChange = (e: ChangeEvent<HTMLInputElement>) => {
```